_posts/convars/2022-02-02-mom_hud_status_enable.md

```diff
@@ -10,4 +10,4 @@ maximum_value: 1
 default_value: 1
 ---
 
-Toggles showing the map name.
\ No newline at end of file
+Toggles showing hud status.
```

Comment:
The old description was quite vague. Would be good to specify what exactly is the status HUD.
```suggestion
Toggles showing hud status, which displays information about which zone you are in, which saveloc was last loaded, whether you are in practice mode, etc.
```